src/sage/rings/polynomial/laurent_polynomial.pyx

```diff
@@ -1318,22 +1318,59 @@ cdef class LaurentPolynomial_univariate(LaurentPolynomial):
             return ~self
         raise ArithmeticError("element is not a unit")
 
+    @coerce_binop
     def xgcd(self, other):
-        """
+        r"""
         Extended `gcd` for univariate Laurent polynomial rings over a field.
```

Comment:
Maybe it was my fault to use \`gcd\` instead of gcd, which looks better

Reply:
Shall I change it then?

Reply:
It is not important, but either gcd or \`\gcd\` will look better. Otherwise, LGTM

Reply:
Thanks. I opted to link to the method.